Amplify/Categories/Auth/Models/Options/AWSAttributeResendConfirmationCodeOptions.swift

```diff
@@ -0,0 +1,24 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+/// When you use the ResendConfirmationCode API action, Amazon Cognito invokes the function that is assigned to the custom message trigger.
```

Comment:
This comment should be for the `metadata` right and not for `AWSAttributeResendConfirmationCodeOptions`